docs/runbook-ecir2019-ccrf.md

```diff
@@ -65,17 +65,17 @@ Run the following commands to index the `Robust04`, `Robust05`, and `Core17` col
 
 ```bash
 nohup sh target/appassembler/bin/IndexCollection -collection TrecCollection \
- -generator JsoupGenerator -threads 16 -input /path/to/robust04 \
```

Comment:
hrm, I'm torn about this.

The `runbook-*` files are really runbooks, so it's implicit that they only work with the commit at the time they were written... 

So if we roll back to an old commit, the runbooks should still work.

I'm worried that if we partially fix the runbook, then we'll be in a weird broken state...

thoughts?

Reply:
I was also a bit concerning about the changes to runbooks. IMHO it would be nice to note in the runbook which commit specifically is the runbook validated against. May be this will be the commit before `JsoupGenerator` is removed?

Perhaps I should leave out the changes to runbooks for now? And maybe we can create a ticket for validating and stamping the runbook with a given *working* commit, then changes to the runbook can be restricted.